tests/functional/modules/test_zos_fetch_func.py

```diff
@@ -916,3 +916,44 @@ def test_fetch_gdg(ansible_zos_module):
 
         if os.path.exists(dest_path):
             shutil.rmtree(dest_path)
+
+
+@pytest.mark.parametrize("relative_path", ["tmp/", ".", "../tmp/", "~/tmp/"])
+def test_fetch_uss_file_relative_path_not_present_on_local_machine(ansible_zos_module, relative_path):
+    hosts = ansible_zos_module
+    current_working_directory = os.getcwd()
+    src = "/etc/profile"
+
+    # If the test suite is running on on root to avoid an issue we check the current directory
```

Comment:
```suggestion
    # If the test suite is running on root to avoid an issue we check the current directory
    # Also, user can run the tests from ibm_zos_core or tests folder, so this will give us the absolute path of our working dir.
```